### PR TITLE
feat: change EXPIRE WHEN to EXPIRE AFTER

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

Change expire declaration gramma in `CREATE FLOW` from `EXPIRE WHEN` to `EXPIRE AFTER`

Related proto PR https://github.com/GreptimeTeam/greptime-proto/pull/170

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

### CODE REVIEW

**Review by fengjiachun — APPROVED** [review]
LGTM

**Review by discord9 — APPROVED** [review]

File: src/common/meta/src/key/flow/flow_info.rs [hunk 1]
```diff
@@ -123,7 +123,8 @@ pub struct FlowInfoValue {
     /// The raw sql.
     pub(crate) raw_sql: String,
     /// The expr of expire.
-    pub(crate) expire_when: String,
+    /// Duration in second as `i64`.
```
Comment (fengjiachun) [inline src/common/meta/src/key/flow/flow_info.rs:126]:
```suggestion
    /// Duration in seconds as `i64`.
```

File: src/flow/src/adapter/worker.rs [hunk 2]
```diff
@@ -232,12 +232,12 @@ impl<'s> Worker<'s> {
...
         // TODO(discord9): set expire duration for all arrangement and compare to sys timestamp instead
-        expire_when: Option<repr::Duration>,
-        create_if_not_exist: bool,
+        expire_after: Option<repr::Duration>,
+        create_if_not_exists: bool,
         err_collector: ErrCollector,
     ) -> Result<Option<FlowId>, Error> {
         let already_exist = self.task_states.contains_key(&flow_id);
-        match (already_exist, create_if_not_exist) {
+        match (already_exist, create_if_not_exists) {
```
Comment (fengjiachun) [inline src/flow/src/adapter/worker.rs:240]:
```suggestion
        match (already_exists, create_if_not_exists) {
```

File: src/flow/src/adapter/worker.rs [hunk 3]
```diff
@@ -232,12 +232,12 @@ impl<'s> Worker<'s> {
         source_ids: &[GlobalId],
         src_recvs: Vec<broadcast::Receiver<DiffRow>>,
         // TODO(discord9): set expire duration for all arrangement and compare to sys timestamp instead
-        expire_when: Option<repr::Duration>,
-        create_if_not_exist: bool,
+        expire_after: Option<repr::Duration>,
+        create_if_not_exists: bool,
         err_collector: ErrCollector,
     ) -> Result<Option<FlowId>, Error> {
         let already_exist = self.task_states.contains_key(&flow_id);
```
Comment (fengjiachun) [inline src/flow/src/adapter/worker.rs:239]:
```suggestion
        let already_exists = self.task_states.contains_key(&flow_id);
```

**Review by fengjiachun — APPROVED** [review]
Rest LGTM

**Pull request merged**